Sources/XcbeautifyLib/String+CapturedGroups.swift

```diff
@@ -11,7 +11,7 @@ extension String {
             return results
         }
 
-        let matches = regex.matches(in: self, range: NSRange(location:0, length: self.count))
+        let matches = regex.matches(in: self, range: NSRange(location:0, length: self.utf16.count))
```

Comment:
Why do you need UTF16View's count?

Reply:
To be Unicode-safe. `NSRegularExpression` ingests UTF-16 code units—the "NS" class prefix hints towards this.